packages/astro/package.json

```diff
@@ -0,0 +1,136 @@
+    "lint": "eslint src/",
+    "lint:attw": "attw --pack .",
+    "lint:publint": "publint",
+    "publish:local": "npx yalc push --replace --sig"
+  },
+  "files": [
+    "dist",
+    "bundled",
+    "client",
+    "server",
+    "internal",
+    "components",
+    "env.d.ts"
+  ],
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js"
+    },
+    "./client/react": {
+      "types": "./dist/client/react/index.d.ts",
+      "import": "./dist/client/react/index.js"
+    },
+    "./client/stores": {
+      "types": "./dist/client/stores/index.d.ts",
+      "import": "./dist/client/stores/index.js"
+    },
+    "./client": {
+      "types": "./dist/client/index.d.ts",
+      "import": "./dist/client/index.js"
+    },
+    "./client/bundled": {
+      "types": "./dist/client/bundled.d.ts",
+      "import": "./dist/client/bundled.js"
+    },
+    "./integration": {
+      "types": "./dist/integration/index.d.ts",
+      "import": "./dist/integration/index.js"
+    },
+    "./integration/bundled": {
+      "types": "./dist/integration/bundled.d.ts",
+      "import": "./dist/integration/bundled.js"
+    },
+    "./internal": {
+      "types": "./dist/internal/index.d.ts",
+      "import": "./dist/internal/index.js"
+    },
+    "./internal/bundled": {
+      "types": "./dist/internal/bundled.d.ts",
+      "import": "./dist/internal/bundled.js"
+    },
+    "./server": {
+      "types": "./dist/server/index.d.ts",
+      "import": "./dist/server/index.js"
+    },
+    "./bundled": {
+      "types": "./dist/bundled.d.ts",
+      "import": "./dist/bundled.js"
+    },
+    "./env": "./env.d.ts",
+    "./components/control": "./components/control/index.ts",
+    "./components/control/*": "./components/control/*",
+    "./components/react": "./components/react/index.ts",
+    "./components/react/*": "./components/react/*",
+    "./components/interactive": "./components/interactive/index.ts",
+    "./components/interactive/*": "./components/interactive/*",
+    "./components/*": "./components/*",
+    "./package.json": "./package.json"
+  },
+  "imports": {
+    "#async-local-storage": {
+      "workerd": "./dist/async-local-storage.server.js",
+      "browser": "./dist/async-local-storage.client.js",
+      "node": "./dist/async-local-storage.server.js",
+      "default": "./dist/async-local-storage.server.js"
+    }
+  },
+  "dependencies": {
+    "@clerk/backend": "1.2.4",
+    "@clerk/clerk-js": "5.8.0",
+    "@clerk/shared": "2.3.2",
+    "@clerk/types": "4.7.0",
+    "@nanostores/react": "0.7.2",
+    "nanoid": "5.0.7",
+    "nanostores": "0.10.3",
+    "path-to-regexp": "6.2.2"
+  },
+  "devDependencies": {
+    "@clerk/eslint-config-custom": "*",
+    "react": "*",
+    "typescript": "*"
+  },
+  "peerDependencies": {
+    "astro": "^3.2.0 || ^4.0.0"
+  },
+  "engines": {
+    "node": ">=18.17.0"
```

Comment:
The README says that Node 20 is required?

Reply:
I had it original to node 20, but because our ci runs with v18 i had to update it. No worries i've checked the Astro minimum node version and this is compatible, also the API for `AsyncLocalStorage` that we are using has been stable for many versions prior to v18.